webserver/static/scripts/similarity/eval-recording.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+
+function EvalRecording(props) {
```

Comment:
see if you can write all your new react components as classes

Reply:
I'll leave this one for now as other people might be looking over this evaluation frontend. But now, with the use of react hooks, the functional components are more in-style than ever!